src/Query/RetryExecutor.php

```diff
@@ -29,12 +31,12 @@ public function tryQuery(Query $query, $retries)
             }
         });
 
+        $executor = $this->executor;
         $success = function ($value) use ($deferred, &$errorback) {
             $errorback = null;
             $deferred->resolve($value);
         };
 
-        $executor = $this->executor;
```

Comment:
Why did you move this line, seems not necessary or am I wrong?

Reply:
yes, not necessary, I will revert it